scala/lambdas/ingest-asset-opex-creator/src/main/scala/uk/gov/nationalarchives/ingestassetopexcreator/Lambda.scala

```diff
@@ -99,7 +104,9 @@ class Lambda extends LambdaRunner[Input, Unit, Config, Dependencies] {
 
 object Lambda {
 
-  case class Input(id: UUID, batchId: String, executionName: String)
+  case class InputItem(id: UUID, batchId: String, assetExists: Boolean)
```

Comment:
Maybe `InputAsset`?

Reply:
Yep ok, it matches the other change now.